### ISSUE TITLE
Remove xdeltaUI from source code

### ISSUE THREAD

**Issue by Miepee** [DodoBirby/MTTNE-Randomiser#2]
xdeltaUI is a piece of software with
a) no license to accompany which details whether redistribution is allowed
b) proprietary, which means no one can audit if the gui tool does anything else

I'd recommend either putting in the readme to suggest downloading an xdelta patcher, or modifying the randomizer to patch the exe itself instead of relying on an external tool.

**Comment by DodoBirby** [comment]
Yeah that makes sense, will probably add a patch function to the randomiser since it's just changing some bytes.
